twoliter/src/lock.rs

```diff
@@ -252,49 +258,38 @@ impl OCIArchive {
 }
 
 /// Represents the structure of a `Twoliter.lock` lock file.
-#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Ord, PartialOrd, Serialize, Deserialize)]
 #[serde(rename_all = "kebab-case")]
 pub(crate) struct Lock {
     /// The version of the Twoliter.toml this was generated from
     pub schema_version: SchemaVersion<1>,
-    /// The workspace release version
-    pub release_version: String,
     /// The resolved bottlerocket sdk
```

Comment:
No not really the schema version here is more to track the schema version of Twoliter.toml

Reply:
I'm imagining that it could also be a problem if you update twoliter and the lockfile no longer matches, despite the contents being the same. I believe that shouldn't be the case here, though.